SimPEG/VRM/ProblemVRM.py

```diff
@@ -13,71 +15,54 @@ class Problem_BaseVRM(Problem.BaseProblem):
 
     """
 
-    # SET CLASS ATTRIBUTES
-    _refFact = None
-    _refRadius = None
-    _indActive = None
     _AisSet = False
+    ref_factor = properties.Integer('Sensitivity refinement factor', min=0)
+    ref_radius = properties.Array('Sensitivity refinement radii from sources', dtype=float)
+    indActive = properties.Array('Topography active cells', dtype=bool)
```

Comment:
This is another very nitpicking comment from me, you can safely ignore it. I was just wondering why `_refFact` is renamed to `ref_factor`, `_refRadius` to `ref_radius`, but `_indActive` to `indActive` and not to `ind_active`?

Reply:
Good point. From my understanding there was standard way for naming classes, methods, properties and variables. I can add this to the issues related to VRM which will address everything in another pull request. Issue #714